### PR TITLE
Vuejs Examples

### PR DESCRIPTION
**Pull request by ambarwootric**
Includes `THREE` vue project each with different use cases of vuejs and setup.
1. `inline-example` using inline vuejs cdn link.
vuejs-inline-example

2. `vue-cli-example` using vue cli command line to create and setup project.
vuejs-vue-cli-example

3. `nuxt-js-example` using nuxt js framework and command line tool to setup project.
vuejs-nuxt-js-example

Each of the projects has there own `README` file on how to run and test projects. Please replace `account_token` with yours.

### CODE REVIEW

File: vuejs/inline-example/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#inline-example
+
```
Comment (hachibu) [inline vuejs/inline-example/README.md]: Add instruction of which file to change account token in.

File: vuejs/nuxt-js-example/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# nuxt-js-example
```
Comment (hachibu) [inline vuejs/nuxt-js-example/README.md]: What is nuxt-js. Delete this?

File: vuejs/vue-cli-example/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# vue-cli-example
+
```
Comment (hachibu) [inline vuejs/vue-cli-example/README.md]: Add instruction of which file to replace account token in.

**Comment by ambarwootric** [comment]
Thanks for review @hachibu I have updated the instructions for all project.

Ray Nuxt.js helps to easily create/manage vue js applications with feature like SSR (server side rendering) etc. Vue js customer also tends to use Nuxt.js for projects and here we have bit different way to add wootric sdk. So please if we can not delete this 🤔
And I am using `nuxt-js-example` folder 😕

File: vuejs/nuxt-js-example/nuxt.config.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    meta: [
+      { charset: 'utf-8' },
+      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
+      { hid: 'description', name: 'description', content: pkg.description }
+    ],
+    link: [
+      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }
+    ],
+    script: [
+      { src: 'https://cdn.wootric.com/wootric-sdk.js' }
```
Comment (ambarwootric) [inline vuejs/nuxt-js-example/nuxt.config.js]: Ray I am using our `wootric-sdk` here in `nuxt-js-example` project.

**Pull request merged**